npm/post-install.js

```diff
@@ -0,0 +1,30 @@
+// @ts-check
+import { family, GLIBC, MUSL } from "detect-libc"
+import { exec } from 'child_process'
+import util from 'util'
+
+const execa = util.promisify(exec)
+const platform = process.platform
+const arch = process.arch
+
+let libcFamily
+family().then((fam) => {
+  libcFamily = fam
+})
+
+let libc
+if (platform === "win32") {
+  libc = "-msvc"
```

Comment:
Why is this needed?

Reply:
Well we have a "gnu" version of windows build so I hardcoded it for now since it also works for win32 arm64.
As for programmatically detecting it I couldn't find anything concrete and it seemed to involve writing native C code to use the relevant APIs.